plugins/chat/assets/javascripts/discourse/components/chat-composer-dropdown.js

```diff
@@ -1,68 +1,10 @@
 import Component from "@glimmer/component";
-import { iconHTML } from "discourse-common/lib/icon-library";
-import tippy from "tippy.js";
 import { action } from "@ember/object";
-import { hideOnEscapePlugin } from "discourse/lib/d-popover";
-import { tracked } from "@glimmer/tracking";
 
 export default class ChatComposerDropdown extends Component {
-  @tracked isExpanded = false;
-  @tracked tippyInstance = null;
-
-  trigger = null;
-
   @action
-  setupTrigger(element) {
-    this.trigger = element;
-  }
-
-  get ariaControls() {
-    return this.tippyInstance?.popper?.id;
-  }
-
-  @action
-  toggleExpand() {
-    if (this.args.hasActivePanel) {
-      this.args.onCloseActivePanel?.();
-    } else {
-      this.isExpanded = !this.isExpanded;
-    }
-  }
-
-  @action
-  onButtonClick(button) {
-    this.tippyInstance.hide();
+  onButtonClick(button, closeFn) {
```

Comment:
On UI usage...I do find it a bit weird the chat composer menu goes to the right and covers the composer. IMO it looked nicer when it went above and had the arrow:

![image](https://github.com/discourse/discourse/assets/920448/b273af04-1851-487f-9875-65922132ce78)

Reply:
Yes agreed, I can change the options for this one :+1: